pkg/services/rpcsrv/server_test.go

```diff
@@ -2596,6 +2799,77 @@ func testRPCProtocol(t *testing.T, doRPCCall func(string, string, *testing.T) []
 			checkContract(t, verAcc, invocScript, 146960) // No C# match, but we believe it's OK and it has a specific invocation script overriding anything server-side.
 		})
 	})
+	t.Run("sendrawtransaction too big", func(t *testing.T) {
```

Comment:
Create a single testcase named "sendrawtransaction". Put all `sendrawtransaction`-related testcases into it: testcase named "invalid size", testcase named "mempool OOM", etc.

Reply:
done